internal/learn/future/shimmer.md

```diff
@@ -0,0 +1,34 @@
+---
+id: shimmer
+title: Shimmer
+description: Introduction to the emerging incentivized staging network
+keywords:
+  - Incentivized
+  - staking
+  - SMR
+---
+
+IOTA Foundation [introduced](https://blog.iota.org/introducing-iota-staking/) Shimmer. An incentivized staging network.
+It can be used to work with upcoming IOTA updates before they arrive on the mainnet. It will also have its own token called [SMR](https://shimmer.network/token).
+To fairly distribute tokens on the new network, you can stake your IOTA tokens before launch.
+
+## Pre-launch staking
+
+Before the Shimmer network goes online, the IOTA Foundation will announce a start date for "pre-launch" staking.
+You will be able to make a transaction to yourself with a certain payload to declare that you want to stake that amount of IOTA tokens to receive SMR. Staking will now be possible for 90 days. You will earn 1 SMR for every 1Mi every 10 seconds.
+There is no risk in staking your IOTA Tokens.
+Let's take a closer (simplified) look at the technology behind the process.
+
+### Technical overview
+
```

Comment:
Most newcomers won't know what Hornet is, so maybe we need to explain this at a more abstract level first and move the specifics more down towards the end?

Reply:
Hmm towards the end of what? The paragraph or the article?

Reply:
As in have the first part with the introduction to Shimmer and the example of Bob using his wallet to stake and after that go into specifics like the Hornet plugin, does that make sense? I think we could make the distinction between what a user needs to do and what the nodes will do for them a bit more clear.

Reply:
But I use the example to better explain the plugin too. So if I move the example up no one will get what the plugin is 😅

Reply:
I see what you mean, one sec.